app/templates/frameworks/dashboard.html

```diff
@@ -20,17 +20,25 @@
 {% endblock %}
 
 {% block main_content %}
-
-  {% with
-     heading = "Apply to G-Cloud 7",
-     smaller = True,
-     with_breadcrumb = True
-  %}
-    {% include "toolkit/page-heading.html" %}
-  {% endwith %}
-
+  {% if g7_status == 'open' %}
+    {% with
+       heading = "Apply to G-Cloud 7",
+       smaller = True,
+       with_breadcrumb = True
+    %}
+      {% include "toolkit/page-heading.html" %}
+    {% endwith %}
+  {% else %}
+    {% with
+      heading = "Your G-Cloud 7 application",
+      smaller = True,
+      with_breadcrumb = True
+    %}
+      {% include "toolkit/page-heading.html" %}
+    {% endwith %}
+  {% endif %}
   <aside role="complementary" class="framework-application-status">
-    Deadline <strong>{{ deadline|safe }}</strong>
+    Deadline {% if g7_status == 'pending' %} for submissions passed at {% endif %} <strong>{{ deadline|safe }}</strong>
```

Comment:
Is this correct? I thought pending was the state the framework was in before it went live. I remember discussing framework statuses with @cathrooney but I can't remember the details.